actionview/lib/action_view/context.rb

```diff
@@ -13,6 +13,7 @@ module ActionView
   # defined below).
   module Context
     attr_accessor :output_buffer, :view_flow
+    attr_reader :current_template
```

Comment:
Doesn't seem right to me to expose the current template here. `_run` from ActionView::Base assigns `@current_template`, I think that's what @tenderlove meant to reuse.